exporters/geneva/include/opentelemetry/exporters/geneva/metrics/exporter.h

```diff
@@ -65,13 +68,8 @@ class Exporter final : public opentelemetry::sdk::metrics::PushMetricExporter {
   std::unique_ptr<DataTransport> data_transport_;
 
   // metrics storage
-  char buffer_non_histogram_[kBufferSize];
-  char buffer_histogram_[kBufferSize];
-  uint64_t buffer_index_non_histogram_;
-  uint64_t buffer_index_histogram_;
+  char buffer_[kBufferSize];
```

Comment:
As the buffer unified, should the buffer size be increased?

Reply:
We always export one metric point at given time, so this buffer will either contain histogram metric point, or non-histogram metric point at given time.  Earlier, we had two buffers as it allowed us to initialize static data (account name, and namespace) for both types of metrics during bootstrapping. As the data is no longer always static, we get rid of two buffers, and only used one.